### ISSUE TITLE
[a11y] User denied permission message should be user friendly

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1405]
![image](https://user-images.githubusercontent.com/37256050/167163853-9cad05f5-e56c-4918-8da7-0f570a58c3ff.png)

Current message when trying to access the security panel
